index.html

```diff
@@ -1208,6 +1205,10 @@ <h4 id="signed-raw">Signed Raw Transaction</h4>
       // Sign and broadcast!
       if (!masterKey.isHardwareWallet) {
         const sign = await cTx.sign(masterKey, 1);
+        const futureTxid=swapHEXEndian(await hash(Crypto.util.hexToBytes((await hash((Crypto.util.hexToBytes(sign)))))));
+  
+        mempool.addUTXO(futureTxid,"",nChange,"",0,"",Mempool.PENDING)
+        mempool.addUTXO(futureTxid,"",nValue,"",1,"",Mempool.PENDING)
```

Comment:
I think this could lead to some errors (and also could be simplified a lot).

For example, if `sendTransaction()` fails for whatever reason, we (above) already assume the TX is in the mempool, so our UTXOs would "double".

Instead, I think we could make `sendTransaction` asynchronous, return the TXID from Blockbook, and then add the resulting TXID via `addUTXO`, this avoids errors and also removes having to hash the TXID ourselves in-advance.

Reply:
Yeah nice idea, I just don't understand why we should get the id from blockbook if we already have it, for the moment I will only add the change if the transaction returns true